src/main/java/duke/EmptyCommandException.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+/**
+ * Represents the empty command exception when nothing is feed into the command line.
+ */
+public class EmptyCommandException extends Exception {
```

Comment:
I like how you have different exceptions and custom error messages for each - really improves the user experience!